local/gugcat/renderer.php

```diff
@@ -229,8 +229,8 @@ public function display_aggregation_tool($rows, $activities) {
             $htmlrows .= html_writer::tag('td', html_writer::tag('a', null, array('class' => $classname, 'href' => $requireresiturl)));
             $htmlrows .= html_writer::tag('td', $row->completed);
             $htmlrows .= ($row->aggregatedgrade->display != get_string('missinggrade', 'local_gugcat')) 
-            ? html_writer::start_tag('td').$row->aggregatedgrade->display.$this->context_actions($row->studentno, null, true, $courseformhistoryparams, true).html_writer::end_tag('td')
-            : html_writer::tag('td', $row->aggregatedgrade->display);
+            ? html_writer::start_tag('td').str_replace(")", "0)", $row->aggregatedgrade->display).$this->context_actions($row->studentno, null, true, $courseformhistoryparams, true).html_writer::end_tag('td')
+            : html_writer::tag('td', str_replace(")", "0)", $row->aggregatedgrade->display));
```

Comment:
This is not the best fix. Just change the decimal place to 3 instead of 2 in number_format() for aggregatedgrade->display in grade aggregation class. 
![image](https://user-images.githubusercontent.com/74167293/106842587-449fde00-66df-11eb-9d3f-aba5f48a2fbf.png)